mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -15,6 +22,8 @@ pub struct BridgeSettings {
     remotes: Vec<ConnectionSettings>,
 
     messages: MessagesSettings,
+
+    storage: Option<StorageSettings>,
```

Comment:
Isn't it should be for ring buffer by default?

Reply:
I was planning on not having the default for RingBuffer here, rather in the bridge creation. So, if there are no settings for storage passed into the bridge, then it defaults to RingBuffer. That sound okay?

Reply:
Just added to defaults.json

Reply:
Found that adding to defaults.json results in a lot of problems as the data being represented is an enum. So, during config merge it can end up trying to put memory and ring_buffer, which is obviously bad... so have opted for a default on StorageSettings and setting it in deserialize if no other version is found. 

If there is a better way to make this work, please let me know!